app/assets/stylesheets/mobile/comments.scss

```diff
@@ -48,4 +50,24 @@
       &.entypo-heart.active { color: $red; }
     }
   }
+
+  .add-comment-switcher { padding-top: 10px; }
+
+  &.inactive { .post_stats, .comment-container { display: none; } }
+}
+
+.stream_element .comments {
+  width: 100%;
+  margin: 0;
+  top: -7px;
+  .content {
+    padding: 0;
+  }
+
+  .comment {
+    padding: 10px 0 0 0;
+    border-top: 1px solid #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.